src/main/scala/org/broadinstitute/dsde/vault/model/UBamCollection.scala

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.dsde.vault.model
+
+import com.wordnik.swagger.annotations.ApiModelProperty
+import spray.json
+
+import scala.annotation.meta.field
+
+object uBAMCollectionJsonProtocol extends json.DefaultJsonProtocol {
+  implicit val impUBamCollection = jsonFormat3(UBamCollection)
+  implicit val impUBamCollectionIngest = jsonFormat2(UBamCollectionIngest)
+  implicit val impUBamCollectionIngestResponse = jsonFormat3(UBamCollectionIngestResponse)
+}
+
+case class UBamCollection
+(
+  @(ApiModelProperty@field)(value = "The Vault ID of this uBAM collection", required = true)
+  id: Option[String] = None,
+  @(ApiModelProperty@field)(value = "The Vault IDs of the uBAMs included in this collection.", required = true)
+  members: Option[Seq[String]] = None,
+  @(ApiModelProperty@field)(value = "The metadata key-value pairs associated with this uBAM collection.", required = true)
+  metadata: Map[String, String]
```

Comment:
1. The `properties` are not being captured / passed on from DM.
2. Do not put default parameters before required parameters.

The same applies for the other two classes.